src/main.rs

```diff
@@ -1,14 +1,38 @@
 extern crate libc;
 use libc::c_char;
 use libc::c_int;
+use libc::c_void;
 use std::ffi::CStr;
 use std::ffi::CString;
+use std::ptr;
+
+#[repr(C)]
+struct deno_buf {
+    data: *const c_char,
+    len: c_int, // TODO(ry) should be size_t.
+}
+
+#[repr(C)]
+struct DenoC {
+    _unused: [u8; 0],
```

Comment:
What's this?

Reply:
This is a how you do an anonymous struct. In C this would be

‘struct DenoC;’